src/server/api/campaign.js

```diff
@@ -158,7 +158,11 @@ export const resolvers = {
     hasUnsentInitialMessages: async (campaign) => {
       const contacts = await r.knex('campaign_contact')
         .select('id')
-        .where({ campaign_id: campaign.id, message_status: 'needsMessage' })
+        .where({
```

Comment:
Actually, can we make this a count instead of getting all the ids?
It should be
`const count = await r.getCount(r.knex.....where(...))`